packages/react-dom/src/client/ReactDOM.js

```diff
@@ -107,7 +107,7 @@ setBatchingImplementation(
 
 function createPortal(
   children: ReactNodeList,
-  container: Container,
+  container: Element | DocumentFragment,
   key: ?string = null,
 ): React$Portal {
   if (!isValidContainer(container)) {
```

Comment:
We have this helper function but I think it doesn't make sense. We should test these independently since they're not all the same.

However, should we also make it a DEV only warning so we don't need the code?